bindings/python/examples/exchange/1_create_account.py

```diff
@@ -24,19 +24,17 @@
 secret_manager = StrongholdSecretManager(
     os.environ.get('STRONGHOLD_SNAPSHOT_PATH'), os.environ['STRONGHOLD_PASSWORD'])
 
-wallet = Wallet(os.environ.get('WALLET_DB_PATH'),
-                client_options, CoinType.IOTA, secret_manager)
-
 # Store the mnemonic in the Stronghold snapshot, this only needs to be
 # done once.
-wallet.store_mnemonic(
-    os.environ['MNEMONIC'])
+SecretManager(secret_manager).store_mnemonic(os.environ['MNEMONIC'])
 
-account = wallet.create_account('Alice')
+bib_path = Bip44(
+    coin_type=CoinType.SHIMMER
+)
+wallet_options = WalletOptions(None, None, bib_path, client_options, secret_manager, os.environ.get('WALLET_DB_PATH'))
+wallet = Wallet(wallet_options)
 
 # Set sync_only_most_basic_outputs to True if not interested in outputs that are timelocked,
 # have a storage deposit return, expiration or are nft/account/foundry outputs.
-account.set_default_sync_options(
+wallet.set_default_sync_options(
     SyncOptions(sync_only_most_basic_outputs=True))
-
-print(account.get_metadata())
```

Comment:
Shouldn't we print at least "something"? You could assign an Wallet alias and then the happe message: `Wallet <xyz> created." or smth.

Reply:
addded the same as the other create_wallet, renamed the file